deploy/cr/tls/cr_minimal_with_tls.yaml

```diff
@@ -7,10 +7,10 @@ spec:
   security:
     endpointEncryption:
             # Define the certificate to encrypt endpoint traffic.
-            # `type: service` uses platform service certificate.
-            # `type: secret` uses custom certificates.
-            type: secret
-            # If `type: service`, uncomment the following `certServiceName` line:
+            # `type: Service` uses platform service certificate.
+            # `type: Secret` uses custom certificates.
+            type: Secret
```

Comment:
@rigazilla can we add validate for incorrect values, i.e. not Service and Secret in some place or it doesn't make sense?

Reply:
@dmvolod  If it sounds good, I would plan a general assessment on yaml input validation for 2.1

Reply:
Yes, sure, @rigazilla good plan.